packages/website/package.json

```diff
@@ -7,10 +7,11 @@
     "private": true,
     "scripts": {
         "test": "jest",
-        "dev": "node generate-pages.js --config=website-dev-config.json && FORCE_EXTRACT_REACT_TYPES=true next dev",
+        "dev": "node start.js --config=website-dev-config.js",
         "build": "next build",
```

Comment:
Does this need to run generate pages first to be valid? Doesn't build need to be run out of our bin?

Similar, instead of having a binary `start.js`, the binary should point to something called `run.js` or something which takes an arg which is the command you are running, so `dev` would be `node run.js start --config=website-dev-config.js`